### ISSUE TITLE
Unfinished Product Disclaimer

### ISSUE THREAD

**Issue by M3TIOR** [GentenStudios/Phoenix#39]
## Story
In case someone decides to distribute the binary when we get a little bigger, and the project is a little more fleshed out, we should have an unfinshed product disclaimer so that people know this is still a work in progress and that we could use help. It would be a good idea to leave a link to the git repo as well.

## MVP
What are the minimum features needed for this story?
[ ] Any *unfinished product disclaimer* either as a splash screen or as an base game element that's front and center.

[ ] Perhaps a video or some kind of artistic rendering that makes the disclaimer more visually appealing to players. For example: A hostile base game mob half drawn chainsing in the background chasing the player with the disclaimer in bold text above that fades out into the
start screen of the game.

**Comment by beeperdeeper089** [comment]
That's probably a good shout, would you say we want to try implement something simple like this into the first release (0.1)?

**Comment by M3TIOR** [comment]
Absolutely. It'd be a great way to generate some contributors and a following, and it'd give our artists / animators something to do so they're not sitting around feeling helpless.

**Comment by tobyplowy** [comment]
I like the artistic part of this all. Tho I do not think that would be the right way to go for now.
I think that this can be added to the 0.1 but then it will have to super simple and bare bones. Just a text in the corner that says "this is indev software" or something like that will do for 0.1. but for later versions I would be interested in making a little short animation typa deal yeah =D

**Comment by apachano** [comment]
What about an about section in the escape menu?

**Comment by M3TIOR** [comment]
That could be a good place for the larger / more verbose body of information about contributing should we decide to put it into the game. Although I think a front and center lable, however brief, is something we can't live without. We could implement that label as a link to the about section.

**Comment by tobyplowy** [comment]
@beeperdeeper089 Do we add this to 0.1? We can add it somewhere on screen or in the window title.
CC@M3TIOR

**Comment by beeperdeeper089** [comment]
If I get the layer stack in then sure, otherwise no real point tbh.

**Comment by M3TIOR** [comment]
I agree with Beeper at this juncture.

**Comment by M3TIOR** [comment]
Modified this a tad
